R/cal-estimate-none.R

```diff
@@ -0,0 +1,150 @@
+#--------------------------------- Methods -------------------------------------
+#------------------------------- >> None  ----------------------------------
+#' Do not calibrate model predictions.
+#' @inheritParams cal_estimate_logistic
+#' @include cal-estimate-utils.R
+#' @param truth The column identifier for the true class results
+#' (that is factor or numeric). This should be an unquoted column name.
+#' @param estimate A vector of column identifiers, or one of `dplyr` selector
+#' functions to choose which variables contains the class probabilities or
+#' numeric predictions. It defaults to the prefix used by tidymodels (`.pred_`).
+#' For classification problems, the order of the identifiers will be considered
+#' the same as the order of the levels of the `truth` variable.
```

Comment:
shouldn't these be inherited from `cal_estimate_logistic()`?
```suggestion
```

Reply:
They should but I override them since they can take any type of prediction.